test/torchaudio_unittest/prototype/ctc_decoder_test.py

```diff
@@ -98,10 +105,28 @@ def test_get_tokens_and_idxs(self):
     @parameterized.expand([(get_asset_path("decoder/tokens.txt"),), (["-", "|", "f", "o", "b", "a", "r"],)])
     def test_index_to_tokens(self, tokens):
         # decoder tokens: '-' '|' 'f' 'o' 'b' 'a' 'r'
-        decoder = self._get_decoder(tokens)
+        decoder = self._get_decoder(tokens=tokens)
 
         idxs = torch.LongTensor((1, 2, 1, 3, 5))
         tokens = decoder.idxs_to_tokens(idxs)
 
         expected_tokens = ["|", "f", "|", "o", "a"]
         self.assertEqual(tokens, expected_tokens)
+
+    def test_download_pretrained_files(self):
```

Comment:
Can you either mock the http request or move this into integration tests directory?
Unit tests should not rely on any external resources. (tests should be runnable without internet connection)